data/systemd/anaconda-nm-disable-autocons-rhel.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 ConditionKernelCommandLine=|ip
 ConditionKernelCommandLine=|inst.ks
-ConditionOSRelease=ID=rhel
+ConditionOSRelease=|ID=rhel
+ConditionOSRelease=|ID=centos
```

Comment:
What about the other RHEL-like distributions? AlmaLinux and so on. Is there a possibility to have a more general way to fix it everywhere?

Reply:
Hm, maybe we could use ID_LIKE exists and ID_LIKE contains "rhel".
I'll see if/how we can express it by systemd Condition.